.gitignore

```diff
@@ -9,7 +9,6 @@ resources
 .project
 .classpath
 .metadata
-.settings
```

Comment:
Not needed I guess?

Reply:
it is needed to have the VS Code task and launch configuration available in the repository

Reply:
oh my bad, I was in another repository in my head. Effectively, not needed